components/tools/OmeroWeb/test/integration/test_metadata.py

```diff
@@ -49,9 +49,10 @@ def test_pixel_size_units(self):
         request_url = reverse('load_metadata_details', args=['image', iid])
         data = {}
         rsp = get(self.django_client, request_url, data, status_code=200)
-        html = rsp.content
+        html = rsp.content.decode("utf-8")
         # Units are µm by default
-        assert "Pixels Size (XYZ) (µm):" in html
+        value = "Pixels Size (XYZ) ("+u"µm):"
+        assert value in html
```

Comment:
Now fails Python 2 with 
```
>       assert "Pixels Size (XYZ) (µm):" in html
E       UnicodeDecodeError: 'ascii' codec can't decode byte 0xc2 in position 19: ordinal not in range(128)
```

Reply:
This was pushed after 02/11 build. The commit has not yet been included. I expect it to pass tomorrow

Reply:
Green today